docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -75,19 +75,21 @@ Sign in to Grafana and navigate to **Administration > Authentication > Configure
    | **Single logout**                     | The SAML single logout feature enables users to log out from all applications associated with the current IdP session established using SAML SSO. For more information, refer to [SAML single logout documentation]]({{< relref "../saml#single-logout" >}}). |
    | **Identity provider initiated login** | Enables users to log in to Grafana directly from the SAML IdP. For more information, refer to [IdP initiated login documentation]({{< relref "../saml#idp-initiated-single-sign-on-sso" >}}).                                                                 |
 
-1. Click **Next: Key and certificate**.
+1. Click **Next: Sign requests**.
 
-### 2. Key and Certificate Section
+### 2. Sign Requests Section
 
-1. Provide a certificate and a private key that will be used by the service provider (Grafana) and the SAML IdP.
+1. In the **Sign requests** field, specify whether you want the outgoing requests to be signed, and, if so, then:
 
-   Use the [PKCS #8](https://en.wikipedia.org/wiki/PKCS_8) format to issue the private key.
+   1. Provide a certificate and a private key that will be used by the service provider (Grafana) and the SAML IdP.
 
-   For more information, refer to an [example on how to generate SAML credentials]({{< relref "../saml#generate-private-key-for-saml-authentication" >}}).
+      Use the [PKCS #8](https://en.wikipedia.org/wiki/PKCS_8) format to issue the private key.
 
-1. In the **Sign requests** field, specify whether you want the outgoing requests to be signed, and, if so, which signature algorithm should be used.
+      For more information, refer to an [example on how to generate SAML credentials]({{< relref "../saml#generate-private-key-for-saml-authentication" >}}).
```

Comment:
I don't see a Generate private key for SAML authentication section in the topic [Configure SAML authentication using the configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/saml/).

Should this link be: /docs/grafana/latest/setup-grafana/configure-security/configure-authentication/saml/#example-of-how-to-generate-saml-credentials?

Reply:
Great catch, I updated the link. Thank you!

Reply:
It seems there is a drift between the [published version](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/saml/#example-of-how-to-generate-saml-credentials) and the [source code](https://github.com/grafana/grafana/blob/main/docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md?plain=1#L147). My guess is that the latest changes from the docs haven't been published. I'm not sure what to do in this case, can you please help?

Reply:
@chri2547 please let me know if you see any issues with this link. In my opinion, on every update to the docs we should always refer the source code instead of the deployed docs in prod.

Reply:
> @linoman , should 89767 be backported?
No. This is a new feature for 11.2.

@dmihai If this docs is a new feature for 11.2, just make sure that this will be deployed for 11.2 or "next" release.

Reply:
@chri2547 can you please help unblock this PR? Both my changes and Lino's refer to new features that will be available in Grafana v11.2. So I think the link I had initially is the correct one to use.